apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -14,22 +14,35 @@ <h1>View Users</h1>
   <hr />
   <p style="margin-bottom: 30px">View submitted users.</p>
   <hr />
-  <div class="filter-container">
+  <div class="filter-container" style="display: flex; gap: 10px;">
     <div class="filter-content">
       <span><b>View Users by Organization: </b></span>
       <select id="organizationFilter" class="organization-filter" onchange="filterTableByOrganization()">
         {% for option in filter_options %}
-          <option class="dropdown-text" {% if option == selected_filter %}selected{% endif %}>{{ option }}</option>
+          <option class="dropdown-text" {% if option == selected_org %}selected{% endif %}>{{ option }}</option>
         {% endfor %}
       </select>
     </div>
+    <div class="filter-content">
+      <span><b>Is Status Active: </b></span>
+      <select id="statusFilter" class="status-filter" onchange="filterTableByStatus()">
+        {% for option in status_options %}
+        <option class="dropdown-text" {% if option == selected_status %}selected{% endif %}>{{ option }}</option>
+      {% endfor %}
+      </select>
+   </div>
+  {% if selected_status or selected_org %}
+        <button onclick="clearSelections()">Clear Options</button>
+      {% endif %}
   </div>
```

Comment:
```suggestion
     </div>
  </div>
```
Related to `filter-content` comment above.